org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/view/PepysImportView.java

```diff
@@ -68,6 +68,7 @@
 	private Button importButton;
 	private Button testConnectionButton;
 	private Button useCurrentViewportButton;
+	private Button clearAreaButton;
```

Comment:
@saulhidalgoaular  - minor question.  This really isn't important, I'm just "thinking out loud".

It caught my eye that these fields weren't final, because  I'm sure the GUI elements won't get deleted/edited.

I looked at the code.  They can't be final because they're not created in the constructor.  But, they are created in a method that only gets called from the constructor.

So, if the UI was actually built in the constructor they could be final.

What do you think about the pros/cons of doing that?

Reply:
Good afternoon @IanMayo 

Very good observation. I consider you are correct, and controls should always be final. In other case, we are in risk of losing the bindings created by the controller.

I will change it and I would also remove access from the view to the model (removing the model from the view constructor), because it is conceptually not very precise.